### PR TITLE
A docker environment for loading xlsx data in database using docker

### PR DESCRIPTION
**Pull request by jhagberg**
Added a small docker-compose file
A small container with everything needed to run the import scripts
Bash files with small changes to run everything using docker-compose
now re-run csv creation if file csv file is of zero bytes

### CODE REVIEW

**Comment by pontus** [comment]
I don't see the specific load scripts? (`load-mellerud.sh` and so on).

File: scripts/load-docker-compose.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+	    -h		Show this help text
+	USAGE_END
+}
+
+
+while getopts 'g:G:hm:' opt; do
+	case $opt in
+		g)	gfile=$OPTARG ;;
+		G)	Gfile=$OPTARG ;;
```
Comment (pontus) [inline scripts/load-docker-compose.sh:33]: Naming? Could this be grabbit and gherd or grdata/ghdata or something?

**Review by pontus — APPROVED** [review]

**Comment by pontus** [comment]
Approved now, but that is more to show that what's here is fine, I still think scripts are missing.

**Comment by jhagberg** [comment]
The missing script is already in the repo I did not need to change them

File: scripts/load-docker-compose.sh (hunk 1 above)
Comment (jhagberg) [reply scripts/load-docker-compose.sh:33]: I just took @kusalananda script so need to update that instead

**Comment by pontus** [comment]
Ah, must have confused branches or looked in an non-pulled repo.

**Pull request merged**